src/controller.coffee

```diff
@@ -224,6 +224,10 @@ exports.Editor = class Editor
 
     # Instantiate an ICE editor view
     @view = new view.View extend_ @standardViewSettings, respectEphemeral: true
+    @paletteView = new view.View extend_ @standardViewSettings, {
+      showDropdowns: @options.showDropdownInPalette ? false
```

Comment:
Ah hah, got it: an option to show dropdowns at all, which is then used when drawing the palette.  Nice approach.